Tests/scripts/validate_files_structure.py

```diff
@@ -51,10 +51,16 @@
 
 # file types regexes
 IMAGE_REGEX = r".*\.png"
-SCRIPT_YML_REGEX = r"{}.*\.yml".format(SCRIPTS_DIR)
-SCRIPT_PY_REGEX = r"{}.*\.py".format(SCRIPTS_DIR)
+SCRIPT_YML_REGEX = r"{}[\\/].*[\\/].*\.yml".format(SCRIPTS_DIR)
+SCRIPT_PY_REGEX = r"{}[\\/].*[\\/].*\.py".format(SCRIPTS_DIR)
+SCRIPT_TEST_DATA_REGEX = r"{}[\\/].*[\\/]test_data[\\/].*".format(SCRIPTS_DIR)
+SCRIPT_TEST_PIP_REGEX = r"{}[\\/].*[\\/]Pipfile(.lock){{0,1}}".format(SCRIPTS_DIR)
 SCRIPT_JS_REGEX = r"{}.*\.js".format(SCRIPTS_DIR)
-INTEGRATION_YML_REGEX = r"{}.*\.yml".format(INTEGRATIONS_DIR)
+INTEGRATION_PACKAGE_TEST_DATA_REGEX = r"{}[\\/].*[\\/]test_data[\\/].*".format(INTEGRATIONS_DIR)
+INTEGRATION_PACKAGE_TEST_PIP_REGEX = r"{}[\\/].*[\\/]Pipfile".format(INTEGRATIONS_DIR)
```

Comment:
Why does it have different format then the Pipfile for scripts

Reply:
it should be the same. We have Pipfile and Pipfile.lock